01week/rockPaperScissors.js

```diff
@@ -37,22 +42,6 @@ function rockPaperScissors(hand1, hand2) {
  
```

Comment:
you could simplify "it's a tie" by sending the condition through the first if statement (hand1 === hand2)